Extensions/xkit_patches.js

```diff
@@ -129,6 +129,52 @@ XKit.extensions.xkit_patches = new Object({
 	patches: {
 		"7.9.0": function() {
 
+			XKit.svc = {
+				blog: {
+					followed_by: data => new Promise((resolve, reject) => {
```

Comment:
let's make Nx_XHR return a promise if there's no `onload` value passed.

Reply:
this would require touching like 3 other extensions, is that okay?

Reply:
Why? Which extensions are currently using Nx_XHR without an `onload` value?

Reply:
ohh you mean a situational promise return. hmmm

Reply:
mmmmmm i dont know how to do this without it being really ugly. i would much rather rewrite it as a promise function entirely either now or later when all Nx_XHR calls are moved to XKit.svc